radish/documentors/__main__.py

```diff
@@ -0,0 +1,211 @@
+        [-o=<folder-path> | --output-folder=<folder-path>]
+        [--index-file-name=<file-path> ]
+        [-s=<file-path> | --single-file=<file-path>]
+        [-c | --print-to-console]
+        [--label-prefix=<label-prefix>]
+        [-r | --recreate]
+        [--no-ansi]
+    radish-docs (-h | --help)
+    radish-docs (-v | --version)
+
+Commands:
+    sphinx
+      create Sphinx documentation
+
+Arguments:
+    modules-path
+      path to python step modules to document
+
+Options:
+    -o=<folder-path>, --output-folder=<folder-path>
+      location of the output folder, by default `_build` in the current working
+      folder
+
+    --index-file-name=<file-name>
+      optional, name for the index file created by `--output-folder`, by
+      default "index.rst". This file is placed in the output folder.
+
+    -s=<file-path>, --single-file=<file-path>
+      write documentation to a single file
+
+    -c, --print-to-console
+      print `--single-file` documentaiton to console
+
+    --label-prefix=<label-prefix>
+      label prefix used for various section of the generated docs. Can be used
+      if you need a unique on in your documentation. By default set to
+      `radish.document_id.`
+
+    -r, --recreate
+      recreate output folder or file by removing and creating it again.
+
+    -h, --help
+      show this screen
+
+    -v, --version
+      show version
+
+(C) Copyright by Timo Furrer <tuxtimo@gmail.com>
+    """
+
+    # enable errororacles except hook system wide
+    enable_excepthook()
+
+    # parse sys.argv against our docopt argments
+    arguments = docopt("radish-docs {0}\n{1}".format(__VERSION__, usage),
+                       version=__VERSION__)
+
+    # create world wide configuration
+    setup_config(arguments)
+
+    # process arguments for "sphinx" command
+    if arguments['sphinx']:
+        # get the module path
+        module_path = arguments['<module-path>']
+
+        # get the output folder or if missing set the default for the
+        # output_folder_path
+        if arguments['--output-folder'] is not None:
+            output_folder_path = arguments['--output-folder']
+        else:
+            output_folder_path = "${PWD}/_build"
+
+        # get the index file name, otherwise default it to index.rst
+        if arguments['--index-file-name'] is not None:
+            index_file_name = arguments['--index-file-name']
+            # test if given file name rather then path, raises RadishError
+            _test_file_name(index_file_name, '--index-file-name')
+        else:
+            index_file_name = "index.rst"
+
+        # set recreate file
+        single_file_path = arguments['--single-file']
+        # set print to console flag
+        print_to_console = arguments['--print-to-console']
+
+        # if console is specified then then set single file to none and
+        # output_folder_path to None, recreate to Flase
+        if print_to_console is True:
```

Comment:
So this is a concern I would like to discuss if you do not mind. I like to avoid *truthiness* style comparissons because I've run into subtle bugs when things are set to things that should return True/False but do not. 

Basically wehn read `if print_to_console` it can translate to any of the following:
- I am checking for empty string
- I am checking for empty list
- I am checking that integer is not 0
- ... and so on ...

If you really want to I will remove it but I'd urge you to consider explicit style checks when the code indicates the exact "thing" it is checking against.